docs/adrs/0077-permanently-store-origin-duty-gbloc.md

```diff
@@ -0,0 +1,107 @@
+
+The downside of using a view to fetch the origin duty location GBLOC is that the data remains dynamic, and there is no way to determine what the *original* GBLOC was if the postal code to GBLOC relationship changes later. Since GBLOCs can be consolidated, USTC needs to be able to access the historical data for moves. 
+
+### Additional considerations behind this change
+
+* View tables are not transferred to data warehouse. USTC needs the duty location GBLOC data warehoused.
+* The `origin_duty_location_to_gbloc` does not handle GBLOC outliers, such as USMC logic, leading to [incorrect data](https://dp3.atlassian.net/browse/MB-15143) :lock:.
+
+### Important distinction to note
+In addition to `origin_duty_location_to_gbloc`, there is another view for `move_to_gbloc`. This is used to fetch the Shipment GBLOC, which is based
+on the origin address of the first shipment in a move. The Shipment GBLOC and Origin Duty Location GBLOC are both fetched, and the API manages the logic
+for which GBLOC to select to route the move to the correct queue. 
+
+For the purpose of the current requirement, the `move_to_gbloc` view will remain in place, and the routing logic in the API will not be changed. The primary change
+will be in storing the origin duty location GBLOC when an MTO is created, and query the stored origin duty location GBLOC.
+
+## Decision Drivers
+
+* Complexity and impact of refactor
+  * What potential does the solution have to introduce breaking changes?
+  * Do we have enough capacity handle the unknowns or deal with future issues this decision may cause?
+  * How disruptive will this work be on other teams?
+* Flexibility
+  * Will this solution support required future changes?
+  * Will it be easy and intuitive to handle GBLOC consolidations and additions?
+* Consistency
+  * Does this change follow best practices?
+  * Does it introduce any anti-patterns?
+
+## Considered Alternatives
+
+> **bold denotes chosen**
+
+* *Do nothing*
+* *Surface Origin GBLOC data in History & Audit Log*
+* **Add a column to the orders table**
+* *Store the Origin Duty Location GBLOC in a join table*
+
+## Decision Outcome
+
+<!-- * Chosen Alternative:  -->
+* Chosen Alternative: *Add a column to the orders table*
+* Positive Outcomes:
+  * Eliminate relying on a view for frequently accessed data
+  * Provides a direct link from an orders record to its GBLOC, which would flow to warehouse
+* Consequences:
+  * Event triggers will need to be updated for history and audit logging purposes
+* Other considerations:
+  * Data backfill will need to be addressed
```

Comment:
How are we addressing this @kctruss do we have tickets in a backlog for this work?

Reply:
I've created subtasks for this work under the current story MB-15032